addons/website/static/src/client_actions/website_preview/website_preview.js

```diff
@@ -435,6 +435,14 @@ export class WebsitePreview extends Component {
         // exist, so we do not replace the iframefallback content.
         // The iframefallback is hidden in test mode
         if (!this.websiteContext.edition && this.iframe.el.contentDocument.body && this.iframefallback.el) {
+            // Remove autoplay in all media video iframes urls so videos are not playing in the background
+            const media_iframes = this.iframe.el.contentDocument.getElementsByClassName("media_iframe_video");
+            for (const media_iframe of media_iframes) {
+                const iframe = media_iframe.getElementsByTagName('iframe')[0];
+                const url = new URL(iframe.src);
+                url.searchParams.delete('autoplay');
+                iframe.src = url.toString();
+            }
```

Comment:
I would have just encapsulated that inside an util method like `_cleanIframeFallback()` or `_neutralizeIframeFallback()` or anything like that
(So you don't have to repeat the whole if condition in method override of `_onPageUnload` and it leaves the original `_onPageUnload` easier to read)